### PR TITLE
Model refactor

### PR DESCRIPTION
**Pull request by MichaelKupriianov**
- Sorting moved to utilities folder
- Bug in TaskManager::Edit method fixed
- TasksSortBy moved to Task.proto
- Code style fixes

### CODE REVIEW

**Comment by KharchenkoMaks** [comment]
> bug in TaskManager::Edit method fixed

maybe it is better to move bug fixing to separate PR?

**Review by antonovchtd — APPROVED** [review]

**Review by antonovchtd — APPROVED** [review]

File: src/utilities/SortingProtoObjects.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include "SortingProtoObjects.h"
+
+namespace model {
+
+bool ComparatorId(const model::TaskWithId& first, const model::TaskWithId& second) {
+    if (first.first < second.first) return true;
```
Comment (AntonApraksin) [inline src/utilities/SortingProtoObjects.cpp]: Maybe `return first.first < second.first;`?

**Review by AntonApraksin — CHANGES_REQUESTED** [review]
Lots of files that are not related to PR

File: src/utilities/SortingProtoObjects.cpp (hunk 1 above)
Comment (MichaelKupriianov) [reply src/utilities/SortingProtoObjects.cpp]: Fixed

**Review by AntonApraksin — APPROVED** [review]

**Pull request merged**